cmc-domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/Claim.java

```diff
@@ -50,6 +50,7 @@ public Claim(
         final LocalDate responseDeadline,
         final boolean moreTimeRequested,
         final String submitterEmail,
+
```

Comment:
can you revert this change? random whitespace changes are usually a no no for PRs unless they are just about that